srcpkgs/python3-pillowfight/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'python3-pillowfight'
+pkgname=python3-pillowfight
+version=0.3.0
+revision=1
+build_style=python3-module
+hostmakedepends="python3-nose"
+makedepends="python3-devel"
+depends="python3-pillow-simd"
+checkdepends="python3-pillow-simd"
+short_desc="Python bindings for libpillowfigh"
+maintainer="idealist1508 <id1508-github@yahoo.com>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/World/OpenPaperwork/libpillowfight"
+python_version=3
+
+_COMMIT="50d965879eb89fdef9be09d6e934329486ff585d"
+distfiles="https://gitlab.gnome.org/World/OpenPaperwork/libpillowfight/-/archive/${_COMMIT}/libpillowfight-${version}.tar.bz2"
```

Comment:
Why not `https://gitlab.gnome.org/World/OpenPaperwork/libpillowfight/-/archive/${version}/libpillowfight-${version}.tar.gz`

Reply:
Same for the others.

Reply:
I used Arch PKGBUILD as a template.